spell/spell-reviewer-goerli-checklist.md

```diff
@@ -18,11 +18,15 @@ Spell Actions:
   * [ ] used in the current spell
   * [ ] cleanup previous ones
   * [ ] if submodule upgrades are present make sure `dss-exec-lib` is synced as well
-* [ ] Rates OK
+* [ ] Static Interfaces
+  * [ ] ensure they match `dss-interfaces`
```

Comment:
'or previous patterns' maybe? I recall Sam used an interface one spell in his own way and I raised a previous pattern.

Reply:
`dss-interfaces` should be the source of truth here, as previous spell might have incorrect interfaces.